### PR TITLE
Updated XMS project's authors and team listings

### PR DESCRIPTION
**Pull request by jsperhac**
## Description

Updated lists of maintainers/contributors (AUTHORS markdown) and personnel and their current locations (team html page). Changed bold tags to strong in accordance with current guidance from CI.

## Motivation and Context

The AUTHORS markdown page dated to 2016, the team page under the About tab was also out of date.

## Tests performed

Viewed files in browser.

## Checklist:

- [X] The pull request description is suitable for a Changelog entry
- [X] The milestone is set correctly on the pull request
- [X] The appropriate labels have been added to the pull request

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -2,22 +2,24 @@
...
 - Joseph White
-- Thomas Yearke
 - Jeanette Sperhac
 - Ryan Rathsam
-- Ben Plessinger
+- Gregary Dean
 
 ## Previous Maintainers
 
+- Steven M. Gallo
```
Comment (jpwhite4) [inline AUTHORS.md:14]: Don't forget Rudra

Comment (jsperhac) [reply AUTHORS.md:14]: Right--thanks for that!

Comment (jsperhac) [reply AUTHORS.md:14]: OK, so I added Rudra and now Shippable tests fail??

Comment (jsperhac) [reply AUTHORS.md:14]: Looks like everything is magically passing (as expected) this morning. Note that in commit 5a52f98, I've removed deprecated html elements and used numbered headings instead, which will incidentally help this page to be interpreted by screen readers.

**Review by jpwhite4 — APPROVED** [review]

**Pull request merged**